app/validators/uniqueness.js

```diff
@@ -0,0 +1,57 @@
+import Ember from "ember";
+import Base from 'ember-validations/validators/base';
+
+export default Base.extend({
+  setupOptions: function() {
+    if(this.options === true) {
+      this.options = {};
+    }
+
+    let options = Ember.$.extend({
+      message: 'is taken.',
+      debounce: 150,
+      type: 'post',
+      data: {},
+    }, this.options);
+
+    this.set('options', options);
```

Comment:
`options` is not observed or used externally here, so making it `this._options` would probably be ideal.